policies/recipes/recipedb.go

```diff
@@ -15,32 +15,85 @@
 package recipes
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"strconv"
 	"strings"
+	"time"
+)
+
+var (
+	dbPath   = "/var/lib/google/osconfig_recipedb"
```

Comment:
Windows path?

Reply:
I added support for Windows. I put it under C:\ProgramData\Google but I am not positive whether that is the correct place or not.